Sources/ProjectSpec/ProjectSpec.swift

```diff
@@ -20,6 +20,9 @@ public struct ProjectSpec {
     public var include: [String] = []
 
     public struct Options {
+        fileprivate static let defaultXcodeVersion = "0910"
+        
+        public var currentXcodeVersion: String
```

Comment:
Can we name this just `xcodeVersion`. You probably just went with the old name, sure why the old property was `currentXcodeVersion`

Reply:
Can we also make this an optional and move the `defaultXcodeVersion` into XcodeGenKit. I'd prefer to keep any hardcoded values out of this module, as some people use it independently and it's really the generator that supports a certain version or not.